src/modelgauge/annotation_pipeline.py

```diff
@@ -112,6 +112,7 @@ def handle_item(self, item: SutInteraction):
 
 class AnnotatorWorkers(CachingPipe):
     def __init__(self, annotators: dict[str, Annotator], workers=None, cache_path=None):
+        self.sleep_time = 10
```

Comment:
Should it be a class-level setting?

Reply:
Probably not, as the use for it is to change one instance for a test.